cmd/merlinagent/main.go

```diff
@@ -35,8 +37,8 @@ import (
 	"github.com/satori/go.uuid"
 	"golang.org/x/net/http2"
 
-	"github.com/ne0nd0g/merlin/pkg/agent"
```

Comment:
I believe best practice is to use the full repo URL. I've updated this upstream to fix the case sensitive error where `ne0nd0g` should have been `Ne0nd0g`. Please put back in the full URL path.

Reply:
After thinking about it more, I see the problem with having the full repo. Kind of a chicken and egg problem. Let me research what the best practice is for this situation.

Reply:
It looks like it is best practice to use the full repo. You would want to `go get github.com/Ne0nd0g/merlin`, rename orgin to upstream, and add your fork as orgin.
https://blog.sgmansfield.com/2016/06/working-with-forks-in-go/
http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html

Reply:
I think we found the same links around the same time! Yeah, you are 100% correct in the way you do it. This is my first go fork and pull request.